src/inference/src/os/win/win_system_conf.cpp

```diff
@@ -204,4 +205,81 @@ int getNumberOfCPUCores(bool bigCoresOnly) {
 std::vector<int> getAvailableNUMANodes() { return {-1}; }
 #endif
 
+std::vector<std::vector<int>> getNumOfAvailableCPUCores() {
```

Comment:
Those features are same implementation on Linux and Windows. Since they are only dependent on _cpu_mapping_table which is common, could we have one implementation in StreamsExecutor?

Reply:
I have moved these features to lin_system_cofig.cpp.